docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Project"
+sidebar_position: 5
+description: >
+  `Project` is **a set of scopes from different domains**, a way to group different resources, and it is crucial for some metric calculation like `DORA`.
+---
+
+## Summary
+`Project` is **a set of scopes from different domains**, a way to group different resources, and it is crucial for some metric calculation like `DORA`.
+
+It contains the following two models:
+ - `projects` describes a project object, including its name, creation and update time and other basic information
+ - `project_metrics` describes the mapping relationship enabled by a plugin, including the name of the project and plugin on both sides of the mapping, and basic information about plugin options.
```

Comment:
`project_metrics` describes what metrics a project had enabled.
"both side of the mapping" doesn't sound right, it feels like you are talking about `project_mapping`, please remove this part to avoid confusion.

Reply:
and i think we should add a new item `project_mapping`